packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -55,7 +55,7 @@ window.\$hotReloadHook = function(modules) {
         // once we've reloaded every module, trigger the hot reload.
         if (reloadCount == modules.length) {
           require(["$entrypoint", "dart_sdk"], function(app, dart_sdk) {
-            window.\$mainEntrypoint = app.main.main;
+            window.\$mainEntrypoint = app[Object.keys(app)[0]].main;
```

Comment:
I'm not really familiar with RequireJS/AMD modules so it's not clear to my why the "app" dependency changed based on the changes in this PR... Is it the case that "app" will only ever have one property but we don't know the name of it at compile time? Please add a comment explaining why it's written this way

Reply:
Added a comment below. Essentially the module name is based off of the file, which for the generated entrypoint is something like `__var__flutter_123abcd__entrypoint`